src/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -478,3 +486,12 @@ process_extending_files()
     done
   done <<<"$(get_matched_files '*.sh' "$@" | sort -u)"
 }
+
+create_extensions()
+{
+  if [ -v POSTGRESQL_EXTENSIONS ]; then
```

Comment:
Without ${POSTGRESQL_EXTENSION}? Is this intended?

Reply:
Yes, I think that's the way -v  work, it need the variable name:
```
 -v varname
              True if the shell variable varname is set (has been assigned a value).
```